app/src/ui/add-repository/write-default-readme.ts

```diff
@@ -5,7 +5,7 @@ const DefaultReadmeName = 'README.md'
 
 function defaultReadmeContents(name: string, description?: string): string {
   return description !== undefined
-    ? `# ${name}\n ${description}\n`
+    ? `# ${name}\n\n${description}\n`
```

Comment:
```suggestion
    ? `# ${name}\n${description}\n`
```

A single newline is consistent with github.com